homeassistant/components/whirlpool/config_flow.py

```diff
@@ -21,6 +23,8 @@
     {vol.Required(CONF_USERNAME): str, vol.Required(CONF_PASSWORD): str}
 )
 
+REAUTH_SCHEMA = vol.Schema({vol.Required(CONF_PASSWORD): str})
```

Comment:
I think the reauth should also show the username field, since that could be the cause of an auth failure (wrong email or typo, for example)

Reply:
This is what the reviewers "required" when I did the Aladdin connect - they said if the user name changes you need to remove and reinstall the integration.  Best I can tell this is how other integrations do it.